### PR TITLE
Fixes Markdown linting issues

### PR DESCRIPTION
**Pull request by gsteel**
|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

Fixes Markdown-lint problems with the docs

### CODE REVIEW

**Comment by Ocramius** [comment]
A `composer install` to force it to generate the plugin map locally is needed

**Comment by gsteel** [comment]
🙄 laminas-cache is too old for 8.0 without `--ignore-platform-reqs`. Update `.laminas-ci.json`? I can do that now…

**Comment by gsteel** [comment]
Gonna have to come back to this later. Removing `laminas-cache` makes installation on 8.0 possible, but psalm complains about the unknown symbols.

_There are zero cache integration tests here_ so all the tests pass happily.

**Comment by Ocramius** [comment]
IMO OK to expand baseline for now.

**Comment by Ocramius** [comment]
Possibly worth declaring a conflict with newer cache releases, if the API is not compatible? :thinking:

**Comment by gsteel** [comment]
> Possibly worth declaring a conflict with newer cache releases, if the API is not compatible?

Installing cache ^3 here appears to be compatible - it's only `Reader` that makes use of a cache (with a static property 🤢).

Docs are out of date, but it should be feasible to add 2||3 to dev deps

File: docs/book/intro.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ In the example below, we demonstrate a simple use case of retrieving an RSS feed
 and saving relevant portions of the feed data to a simple PHP array, which could
 then be used for printing the data, storing to a database, etc.
 
+<!-- markdownlint-disable-next-line heading-increment -->
 > ### RSS optional properties
```
Comment (froschdesign) [inline docs/book/intro.md:22]:
```suggestion
> NOTE: **RSS Optional Properties**
```

File: docs/book/importing.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ to consume it:
 $feed = Laminas\Feed\Reader\Reader::import('http://feeds.example.com/feedName');
 ```
 
+<!-- markdownlint-disable-next-line heading-increment -->
 > ### Importing requires an HTTP client
```
Comment (froschdesign) [inline docs/book/importing.md:12]:
```suggestion
> MISSING: Importing Requires an HTTP Client
```

File: docs/book/find-feeds.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ feeds referenced by a web page with one method call:
 $feedLinks = Laminas\Feed\Reader\Reader::findFeedLinks('http://www.example.com/news.html');
 ```
 
+<!-- markdownlint-disable-next-line heading-increment -->
 > ### Finding feed links requires an HTTP client
```
Comment (froschdesign) [inline docs/book/find-feeds.md:12]:
```suggestion
> MISSING: Finding Feed Links Requires an HTTP Client
```

**Review by froschdesign — CHANGES_REQUESTED** [review]
Admonitions / Call-Outs can be used here and no extra HTML comments / commands are needed.

(The admonitions are managed by the MkDocs plugin [markdown-callouts](https://oprypin.github.io/markdown-callouts/).)

**Review by Ocramius — APPROVED** [review]
Thanks @gsteel!

**Comment by Ocramius** [comment]
@froschdesign argh, 1 minute race condition, sorry!  @_@

**Comment by gsteel** [comment]
Gimme 5 and I'll send in a patch for @froschdesign's suggestions

**Pull request merged**